lib/improver/tests/ensemble_copula_coupling/ensemble_copula_coupling/test_ConvertLocationAndScaleParametersToPercentiles.py

```diff
@@ -112,9 +113,11 @@ def test_check_data(self):
     def test_simple_data_truncnorm_distribution(self):
         """
         Test that the plugin returns an iris.cube.Cube matching the expected
-        data values when a cube containing mean and variance is passed in.
-        The resulting data values are the percentiles, which have been
-        generated using a truncated normal distribution.
+        data values when cubes containing the location parameter and scale
```

Comment:
I guess you could rewrite this test slightly to choose numbers for the loc and scale that aren't equal to the mean and variance to make it clearer what would happen in reality. What do you think?

Reply:
I've made a slight modification. What do you think?

Reply:
Looks better, thank you